src/Orchard.Cms.Web/Modules/Orchard.OpenId/Services/OpenIdService.cs

```diff
@@ -15,31 +15,34 @@ namespace Orchard.OpenId.Services
     public class OpenIdService : IOpenIdService
     {
         private readonly ISiteService _siteService;
-        private readonly IMemoryCache _memoryCache;
         private readonly IStringLocalizer<OpenIdService> T;
-        public OpenIdService(ISiteService siteService, IMemoryCache memoryCache, IStringLocalizer<OpenIdService> stringLocalizer)
+
+        public OpenIdService(ISiteService siteService, IStringLocalizer<OpenIdService> stringLocalizer)
         {
             _siteService = siteService;
-            _memoryCache = memoryCache;
             T = stringLocalizer;
         }
+
         public async Task<OpenIdSettings> GetOpenIdSettingsAsync()
         {
-            var siteSettings = await _siteService.GetSiteSettingsAsync();
-            var result = siteSettings.As<OpenIdSettings>();
+            var settings = await _siteService.GetSiteSettingsAsync();
 
+            var result = settings.As<OpenIdSettings>();
             if (result == null)
             {
                 result = new OpenIdSettings();
             }
+
             return result;
         }
+
         public async Task UpdateOpenIdSettingsAsync(OpenIdSettings openIdSettings)
         {
-            var siteSettings = await _siteService.GetSiteSettingsAsync();
-            siteSettings.Properties[nameof(OpenIdSettings)] = JObject.FromObject(openIdSettings);
-            await _siteService.UpdateSiteSettingsAsync(siteSettings);
+            var settings = await _siteService.GetSiteSettingsAsync();
```

Comment:
> Or is it only a matter of personal taste?

Exactly that. I don't think it's useful to use long variable names when there's 0 ambiguity about the meaning of the represented object.